pkg/proxy/util/utils.go

```diff
@@ -223,3 +229,13 @@ func IsVIPMode(ing v1.LoadBalancerIngress) bool {
 	}
 	return *ing.IPMode == v1.LoadBalancerIPModeVIP
 }
+
+func RetryOnIntr(f func() error) {
```

Comment:
There's already `retry.OnError` from `"k8s.io/client-go/util/retry"`. You should use that instead. (And then you can put the `retriable` helper func for it that compares against `EINTR` in the conntrack code so you don't have to worry about windows.)

Reply:
Thanks for that, I switched to it